### PR TITLE
a30-new-york-times

### PR DESCRIPTION
**Pull request by ghost**
Creation of New York Times

### CODE REVIEW

File: src/jm/html-css/index.html [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>A30 NEW YORK TIMES</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
```
Comment (Iregui44) [inline src/jm/html-css/index.html:11]:
```suggestion
    <link rel="stylesheet" href="styles.css" />
```

File: src/jm/html-css/style.css [hunk 2]
```diff
@@ -0,0 +1,318 @@
+body {
```
Comment (Iregui44) [inline src/jm/html-css/style.css:1]: Chnage name of the file to styles.css

File: src/jm/html-css/style.css [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+}
+
+.button2 {
+  background-color: transparent;
+  text-decoration: none;
+  padding: 3px;
+  border: 1px solid black;
+}
+
+.options {
```
Comment (Iregui44) [inline src/jm/html-css/style.css:61]: I think this menu should cover more of the page.

File: src/jm/html-css/index.html [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+        <button class="button2">Subscribe</button>
+      </div>
+    </header>
+    <hr>
+    <div class="options">
+      <div class="date">
+        <p class="p1"><strong>Thurday</strong></p>
+        <p class="p2">May 21, 2020</p>
+      </div>
+      <ul class="list">
```
Comment (Iregui44) [inline src/jm/html-css/index.html:48]: I think this sould cover more of the page

File: src/jm/html-css/index.html [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+        <li>World</li>
+        <li>Politics</li>
+        <li>Business</li>
+        <li>Opinion</li>
```
Comment (Iregui44) [inline src/jm/html-css/index.html:52]: Aria-label in cbuttons for accesibility

**Comment by Iregui44** [comment]
I dont see changes

File: src/jm/html-css/index.html (hunk 1 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html]: change made

File: src/jm/html-css/style.css (hunk 2 above)
Comment (Juanmediina) [reply src/jm/html-css/style.css]: change made

File: src/jm/html-css/style.css (hunk 3 above)
Comment (Juanmediina) [reply src/jm/html-css/style.css]: ok

File: src/jm/html-css/index.html (hunk 4 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html]: ok

File: src/jm/html-css/index.html (hunk 5 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html]: Aria-label in buttons added

File: src/jm/html-css/index.html [hunk 6]
```diff
@@ -45,21 +45,21 @@ <h1 class="title">THE NEW YORK TIMES</h1>
...
-        <li>Food</li>
-        <li>Travel</li>
-        <li>Magazine</li>
-      </ul>
+      <div class="list">
+        <button aria-label="World">World</button>
+        <button aria-label="Politics">Politics</button>
+        <button aria-label="Business">Business</button>
+        <button aria-label="Opin">Opinion</button>
+        <button aria-label="Tech">Tech</button>
```
Comment (Iregui44) [inline src/jm/html-css/index.html:53]: all buttons most have an href atributee even if its empty with "#"

**Review by Iregui44 — APPROVED** [review]

**Comment by Iregui44** [comment]
You have file conflicts, polease resolve them so I can merge.

**Pull request closed without merging**